tools/apiview/parsers/cpp-api-parser/ApiViewProcessor/AstDumper.cpp

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "AstDumper.hpp"
+#include <iostream>
+#include <list>
+#include <ranges>
+#include <sstream>
+#include <string>
+#include <string_view>
+#include <vector>
+
+constexpr int namespaceIndent = 2;
+
+std::vector<std::string> SplitNamespace(std::string_view const& namespaceName)
+{
+  std::vector<std::string> namespaceComponents;
+#if 0
+  for (const auto ns : std::views::split(namespaceName, std::string_view("::")))
+  {
+    namespaceComponents.push_back(std::string(ns.begin(), ns.end()));
+  }
```

Comment:
Is this coming back?  Can you add comment so others know why its hanging around

Reply:
it's not - std::split doesn't do the right thing(IMHO) yet.